snuba/datasets/configuration/template.yaml

```diff
@@ -0,0 +1,41 @@
+version: v1
```

Comment:
I've added this `template.yaml` file for reference since there isn't a good way to know what a storage config can contain (aside from looking at existing config files), can be removed if deemed unnecessary.